src/builtin-templates/modal.js

```diff
@@ -1,56 +1,57 @@
-import { html, render } from '/web_modules/lit-html/lit-html.js';
-
 export default function create(body, { title = '', isLocked = false } = {}) {
-    const templateResult = wrap(body, title, isLocked);
+    const container = wrap(body, title, isLocked);
 
     return {
-        templateResult,
-        show: () => show(templateResult),
-        fake: () => show(wrap(body, title, isLocked, true)),
+        show({ hidden = false } = {}) {
```

Comment:
show({ hidden }) isn't is a contradiction? :D

Reply:
Heh, yup. I wanted to reduce the API a bit though, and this only really for the 3DS case, so I figured it was the lesser of two evils.